README.md

```diff
@@ -1,8 +1,10 @@
 rust-zonename
 =============
 
-Rust bindings to `getzoneid(3C)`, `getzoneidbyname(3C)`, and
-`getzonenamebyid(3C)` for Illumos based systems.
+Rust bindings to [getzoneid(3C)](https://illumos.org/man/3C/getzoneid),
+[getzoneidbyname(3C)](https://illumos.org/man/3C/getzoneid), and
+[getzonenamebyid(3C)](https://illumos.org/man/3C/getzoneid) for illumos based
+systems.
```

Comment:
at the bottom of the file the function signatures and `errno` crate note should probably be removed

Reply:
Will fix